pkg/symbol/symbolSearcher/symbolSercher.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2022 The Parca Authors
```

Comment:
Also, the directory (unconventional) and file name (typo and conventions) should change `symbolsearcher/seymbol_searcher.go`